### PR TITLE
Create Item Component

### PR DESCRIPTION
**Pull request by Double-A-92**
Created the Item component for solving #9.

Still to do! Maybe create new issues after merging this:
 - Tests
 - Complete copying examples from https://semantic-ui.com/views/item.html --> (Very easy task for Hacktoberfest)

### CODE REVIEW

**Comment by Double-A-92** [comment]
I guess i'll give the tests a try tomorrow...

File: docs/examples/views/CommentExample/Comment.example.vue [hunk 1]
```diff
@@ -74,6 +74,6 @@
 
 <script>
 export default {
-  name: 'CommentExample',
+    name: 'CommentExample',
```
Comment (mariolamacchia) [inline docs/examples/views/CommentExample/Comment.example.vue]: Could you check all the eslint errors? We use 2 spaces :)

File: docs/examples/views/ItemExample/ContentContent.example.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <sui-item>
+      <sui-item-image size="tiny" src="static/images/wireframes/image.png" />
+      <sui-item-content vertical-align="middle">Content C</sui-item-content>
+    </sui-item>
+  </sui-item-group>
+</template>
+
+<script>
+export default {
+    name: 'ContentContentExample',
```
Comment (mariolamacchia) [inline docs/examples/views/ItemExample/ContentContent.example.vue]: Same here, mixed tab length

File: src/views/Item/Item.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { classes, getChildProps, getElementType } from '../../lib';
+
+export default {
+    name: 'SuiItem',
+    description: 'An item view presents large collections of site content for display',
+    props: {
+        href: { type: String, description: 'Specifies a linked document, resource, or location. Only useful if the ItemGroup contains the "link" class.' },
```
Comment (mariolamacchia) [inline src/views/Item/Item.jsx]: I'd break this into multiple lines

File: src/views/Item/ItemContent.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { classes, getChildProps, getElementType } from '../../lib';
+import { Enum } from '../../lib/PropTypes';
+
+export default {
+    name: 'SuiItemContent',
+    description: 'An item can contain content',
+    props: {
+        verticalAlign: Enum(['top', 'middle', 'bottom'], {description: 'Content can specify its vertical alignment'}),
```
Comment (mariolamacchia) [inline src/views/Item/ItemContent.jsx]: I'd break this into multiple lines

File: src/views/Item/ItemContent.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    },
+        render() {
+        const ElementType = getElementType(this);
+        return (
+            <ElementType
+                {...getChildProps(this) }
+                class={classes(
+                    Enum.VerticalAlign.choices.includes(this.verticalAlign) && (this.verticalAlign + ' aligned'),
```
Comment (mariolamacchia) [inline src/views/Item/ItemContent.jsx]: Maybe moving `Enum.VerticalAlign.choices.includes(this.verticalAlign)` to a function would make it more readable.

Also `(this.verticalAlign + ' aligned')` can become `` `${this.verticalAlign} aligner` ``

File: src/views/Item/ItemGroup.jsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+export default {
+    name: 'SuiItemGroup',
+    description: 'A group of items',
+    props: {
+        unstackable: { type: Boolean, description: 'A table can specify how it stacks items responsively' },
+        divided: { type: Boolean, description: 'Items can be divided to better distinguish between grouped content' },
+        relaxed: { type: Boolean, description: 'A group of items can relax its padding to provide more negative space' },
+        veryRelaxed: { type: Boolean, description: 'A group of items can very relax its padding to provide even more negative space' },
+        link: { type: Boolean, description: 'An item can be formatted so that the entire contents link to another page' },
```
Comment (mariolamacchia) [inline src/views/Item/ItemGroup.jsx]: All these should go in multiple lines too

File: src/views/Item/ItemGroup.jsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        const ElementType = getElementType(this);
+        return (
+            <ElementType
+                {...getChildProps(this) }
+                class={classes(
+                    'ui',
+                    this.unstackable && 'unstackable',
+                    this.divided && 'divided',
+                    this.relaxed && 'relaxed',
+                    this.veryRelaxed && !this.relaxed && 'very relaxed',
```
Comment (mariolamacchia) [inline src/views/Item/ItemGroup.jsx]: Maybe it's better to do the opposite. What I mean is that I would expect:
```
<sui-item-group relaxed very-relaxed>...</sui-item-group>
```
to be compiled as 
```
<div class="ui items very relaxed">...</div>
```

File: src/views/Item/ItemHeader.jsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    props: {
+        href: { type: String, description: 'Specifies a linked document, resource, or location' },
+    },
+    render() {
+        const ElementType = (this.href ? 'a' : getElementType(this));
+        return (
+            <ElementType
+                {...getChildProps(this) }
+                href={this.href}
+                class={classes('header', )}
```
Comment (mariolamacchia) [inline src/views/Item/ItemHeader.jsx]: No trailing comma here

File: src/views/Item/ItemImage.jsx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+        href: { type: String, description: 'Specifies a linked document, resource, or location' },
+    },
+    render() {
+        const ElementType = (this.href ? 'a' : getElementType(this));
+        return (
+            <ElementType
+                {...getChildProps(this) }
+                href={this.href}
+                class={classes(
+                    this.size && ('ui ' + this.size),
```
Comment (mariolamacchia) [inline src/views/Item/ItemImage.jsx]: `` `ui ${this.size}` ``

**Comment by mariolamacchia** [comment]
@Double-A-92 let's open separate issues for tests and the missing examples ;) . I'll merge as soon as the comments are addressed

**Comment by mariolamacchia** [comment]
Awesome! Thanks!

**Pull request merged**